charts/tidb-cluster/templates/scripts/_start_scheduled_backup.sh.tpl

```diff
@@ -22,15 +22,14 @@ echo "Increase TiKV GC life time to {{ .Values.scheduledBackup.tikvGCLifeTime |
 /usr/bin/mysql -h${host} -P4000 -u${TIDB_USER} ${password_str} -Nse "update mysql.tidb set variable_value='{{ .Values.scheduledBackup.tikvGCLifeTime | default "720h" }}' where variable_name='tikv_gc_life_time';"
 /usr/bin/mysql -h${host} -P4000 -u${TIDB_USER} ${password_str} -Nse "select variable_name,variable_value from mysql.tidb where variable_name='tikv_gc_life_time';"
 
-/mydumper \
-  --outputdir=${backupPath} \
+/dumpling \
+  --output=${backupPath} \
   --host=${host} \
   --port=4000 \
   --user=${TIDB_USER} \
   --password=${TIDB_PASSWORD} \
-  --long-query-guard=3600 \
-  --tidb-force-priority=LOW_PRIORITY \
-  --regex '^(?!(mysql\.))' \
+  --filter '*.*' \
+  --filter '!mysql.*' \
```

Comment:
We should keep consistent with https://github.com/pingcap/tidb-operator/blob/master/cmd/backup-manager/app/constants/constants.go#L67